packages/jest-enzyme/package.json

```diff
@@ -13,9 +13,10 @@
     "jest": "node ../../node_modules/jest-cli/bin/jest.js __tests__",
     "lint": "eslint ./src",
     "clean": "rm -rf ./lib",
-    "build:js": "babel src --out-dir lib --ignore tests --copy-files",
+    "build:js": "babel --root-mode upward src --out-dir lib --ignore '**/__tests__/*.js' --ignore '**/__failing_tests__/*.js'",
     "build:flow": "flow-copy-source -v -i '**/__tests__/*.js' -i '**/__failing_tests__/*.js' src lib",
-    "build": "npm run clean && npm run build:js && npm run build:flow",
+    "copy-dts": "cpy src/index.d.ts lib",
```

Comment:
https://github.com/babel/babel/issues/6226
> Bug: If in your build script, you have --copy-files passed, files that are ignored in .babelrc files will be copied over, instead of being ignored.